playbooks/aws/openshift-cluster/launch.yml

```diff
@@ -26,12 +26,23 @@
       type: "{{ k8s_type }}"
 
   - include: ../../common/openshift-cluster/set_node_launch_facts_tasks.yml
+    vars:
+      type: "node"
   - include: tasks/launch_instances.yml
     vars:
       instances: "{{ node_names }}"
       cluster: "{{ cluster_id }}"
       type: "{{ k8s_type }}"
```

Comment:
@wshearn what we used openshift-node-{{ node_type }} for the node names instead of openshift-{{ node_type }} ?